Paper.tex

```diff
@@ -1076,7 +1076,7 @@ \subsection{Execution Overview}
 \boldsymbol{\mu}_{\mathbf{m}} & \equiv & (0, 0, ...) \\
 \boldsymbol{\mu}_{\mathrm{i}} & \equiv & 0 \\
 \boldsymbol{\mu}_{\mathbf{s}} & \equiv & () \\
-\boldsymbol{\mu}_{\mathbf{o}} & \equiv & ()
+\boldsymbol{\mu}_{\mathbf{o}} & \equiv & $\varnothing$ \\
```

Comment:
**μ<sub>o</sub>** is different from **o**. The second paragraph of section 9.4 refers to **o**, not **μ<sub>o</sub>**. The latter is the output substate of machine state **μ** and thus should be initialized with an empty sequence, not ∅.